runtime/hsa-runtime/core/driver/xdna/amd_xdna_driver.cpp

```diff
@@ -117,7 +126,10 @@ hsa_status_t XdnaDriver::GetAgentProperties(core::Agent &agent) const {
     return HSA_STATUS_ERROR;
   }
 
-  aie_agent.SetNumCols(aie_metadata.cols);
+  // Right now can only target N-1 columns as that is the
+  // number of shim DMA in Phoenix and Hawkpoint devices
```

Comment:
Need to avoid using codenames.

Reply:
Will we be able to get this information from the driver at some point? It seems error-prone to hardcode things.

Reply:
Changes the naming to be consistent with the xdna-driver. Will add functionality to differentiate between NPU versions where this is not the case.